web/app.yaml

```diff
@@ -32,6 +32,9 @@ handlers:
   - url: /packages/split
     static_dir: packages/split
 
+  - url: /packages/octicons_css
```

Comment:
That's on me. I thought we'd pulled those icons and replaced them with MDC equivalents. I'm also not allowing for primer.css here, which is still in our pubspec.

Any big blockers to removing both (other than the time required to change out the icons)?

Reply:
primer can be removed (https://github.com/dart-lang/dart-pad/issues/1218), and I think this is one of the only cases where we use octicons.